zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2606,6 +2606,42 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /mappings/search:
```

Comment:
❓ in here we are using `/mappings` but in the controller its `mapping-rules`, we should align these, my vote for now would be `/mappings`

Reply:
I'd prefer to keep `mapping-rules` here as its used in the main `MappingController`, as well as in `TenantController`